src/validation.cpp

```diff
@@ -638,7 +642,7 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
                         break;
                     }
                 }
-                if (fReplacementOptOut) {
+                if (fReplacementOptOut || args.disallow_mempool_conflicts) {
```

Comment:
In #22084 I'm updating it to "allow_bip125_replacement" to fix the double-negative and saying "replacements not allowed" instead of "conflicts with mempool." Also moving the `if` condition up as suggested.